Week 15 blogpost (#635)
packit/packit.dev: content/posts/weekly/April-2023.md

---
title: "Packit April 2023"
date: 2023-04-17T15:00:00+01:00
weight: 53
---

## Weeks 14–15 (April 3rd – April 17th)

- `packit source-git` related commands can skip dist-git repos, where the git trailer is not found, when looking for the right dist-git dir where to work. ([packit#1938](https://github.com/packit/packit/pull/1938))

> - Allow running tmt tests with fmf root outside of git root ([packit#1936](https://github.com/packit/packit/pull/1936))

Reviewer: ```suggestion
- Tmt tests can now be run with fmf root outside of git root ([packit#1936](https://github.com/packit/packit/pull/1936))
```

> - Removed adding the "Signed-off-by" tag to commits created by Packit. ([packit#1934](https://github.com/packit/packit/pull/1934))

Reviewer: ```suggestion
- Commits created by Packit no longer contain the "Signed-off-by" tag. ([packit#1934](https://github.com/packit/packit/pull/1934))
```
